src/libostree/ostree-repo-pull.c

```diff
@@ -1598,24 +1597,33 @@ _ostree_repo_sign_verify (OstreeRepo *repo,
         continue;
 
       /* Try to load public key(s) according remote's configuration */
-      if (_load_public_keys (sign, repo, remote_name, &local_error))
+      if (!_load_public_keys (sign, repo, remote_name, error))
+        return FALSE;
```

Comment:
The same idea with multiple signature types here.

Reply:
OK, though I struggle a bit with the fact that as of right now, we have exactly one signing implementation behind this that people should use (they shouldn't use `dummy` of course, in fact I'd like to change it so it requires something like `env OSTREE_TESTSUITE_FLAGS=enable-dummy-sigs ostree pull...` e.g.).

Reply:
hm... I wanted to add ability to switch off the `dummy` engine in a build time.
Please let me know which approach is more preferable?